docs/kusama-parameters.md

```diff
@@ -1,57 +1,74 @@
 ---
-id: kusama-time-periods
-title: Time Period Cheat Sheet
+id: kusama-parameters
+title: Kusama Parameters
 sidebar_label: Parameters
 ---
 
+Many of these parameter values can be updated via the Governance process. If you require absolute
+certainty as to their values, it is recommended you directly check the constants by looking at the
+[chain state](https://polkadot.js.org/apps/#/chainstate/constants) and/or
+[storage](https://polkadot.js.org/apps/#/chainstate).
+
 ### Periods of common actions and attributes
 
-_NOTE: Kusama is 4x as fast as Polkadot will be, except for blocktimes where Polkadot will also have
-6 second blocktimes._
+_NOTE: Kusama generally runs 4x as fast as Polkadot, except Polkadot also has 6 second blocktimes.
```

Comment:
```suggestion
_NOTE: Kusama generally runs 4x as fast as Polkadot, except Polkadot also has 6 second slots.
```

Should we just bite the bullet and commit to calling them 6 second slots instead of block time?

Reply:
yeah